src/ledger.ts

```diff
@@ -177,8 +177,8 @@ export class LedgerCanister {
     requestMemo.setMemo((memo ?? BigInt(0)).toString());
     request.setMemo(requestMemo);
 
-    if (fromSubAccountId !== undefined) {
-      request.setFromSubaccount(subAccountIdToSubaccount(fromSubAccountId));
+    if (fromSubAccount !== undefined) {
+      request.setFromSubaccount(subAccountNumbersToSubaccount(fromSubAccount));
```

Comment:
Looks like a small typo: subAccountNumbersToSub~~a~~**A**ccount

Reply:
They call the class `Subaccount`, therefore I kept it with the same "typo".